src/components/FAQ.js

```diff
@@ -0,0 +1,137 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { makeStyles } from '@material-ui/core/styles'
+import { GatsbyImage, getImage } from 'gatsby-plugin-image'
+import { formatImg } from 'src/utils/formatting'
+import Typography from '@material-ui/core/Typography'
+import faqPattern from 'src/img/causeshared/faqTitlePattern.png'
+import MuiAccordion from '@material-ui/core/Accordion'
+import MuiAccordionSummary from '@material-ui/core/AccordionSummary'
+import MuiAccordionDetails from '@material-ui/core/AccordionDetails'
+import ExpandMoreIcon from '@material-ui/icons/ExpandMore'
+
+const useStyles = makeStyles((theme) => ({
+  parent: {
+    display: 'flex',
+    flexDirection: 'column',
+    width: '100%',
+    background: 'white',
+    alignItems: 'center',
+  },
+  row: {
+    width: '90%',
+    display: 'flex',
+    flexDirection: 'row',
+    alignItems: 'flex-start',
+    marginTop: theme.spacing(8),
+    marginBottom: theme.spacing(8),
+    [theme.breakpoints.down('sm')]: {
+      flexDirection: 'column',
+    },
+  },
+  faqSection: {
+    display: 'flex',
+    flexDirection: 'column',
+    paddingLeft: theme.spacing(3),
+    [theme.breakpoints.down('sm')]: {
+      paddingLeft: theme.spacing(0),
+    },
+  },
+  titleImage: {
+    maxWidth: '35%',
+    [theme.breakpoints.down('sm')]: {
+      maxWidth: 'unset',
+    },
+  },
+  titleAndPattern: {
+    display: 'flex',
+    flexDirection: 'row',
+    justifyContent: 'space-between',
+  },
+  faqPattern: {
+    height: theme.spacing(12),
+  },
+  copy: {
+    marginTop: theme.spacing(3),
+    marginBottom: theme.spacing(3),
+  },
+  titleText: {
```

Comment:
also, we're standardizing our fonts as "variants".  See Theme.js where the caption variant is defined as
```
caption: {
        fontWeight: 500,
        fontFamily: 'Poppins',
        fontStyle: 'normal',
        fontSize: '1rem',
        lineHeight: 1.43,
        color: '#2E282A',
      },
 ```

Reply:
<img width="850" alt="Screen Shot 2022-01-05 at 12 53 17 PM" src="https://user-images.githubusercontent.com/77807117/148287859-74ae732d-0370-4b6c-b4b5-85312c791dc7.png">
You may have seen this already but there's an "inspect" tab on figma which gives exact styles.  Mentioning because fontweight:bold is usually 800 or 900.